src/utilities/common.ts

```diff
@@ -150,6 +150,18 @@ export function fillInFieldConfig (fieldConfig: IFieldConfigPartial): IFieldConf
   };
 }
 
+export function mapFieldSetFields (fieldSet: IFieldSetPartial,
```

Comment:
I was thinking that `fillInFieldSet` is actually a `mapFieldSetFields` with `fillInFieldConfig`, but I didn't want to change something so integral without buy-in

Reply:
DO EEET

Reply:
![image](https://user-images.githubusercontent.com/796717/57032990-a0d7f300-6c19-11e9-8093-af577858856a.png)